src/Faithlife.Data/DbConnectorCommand.cs

```diff
@@ -30,6 +30,11 @@ namespace Faithlife.Data
 		/// </summary>
 		public DbConnector Connector { get; }
 
+		/// <summary>
+		/// True if this command is a stored procedure.
+		/// </summary>
+		public bool IsStoredProcedure { get; }
```

Comment:
I'd prefer `CommandType` be used, here and elsewhere. I agree that there aren't likely to be other command types, but I think the code is also clearer using the enumerated type.